version.props

```diff
@@ -0,0 +1,7 @@
+<!-- This file may be overwritten by automation. Only values allowed here are VersionPrefix and VersionSuffix.  -->
+<Project>
+    <PropertyGroup>
+        <VersionPrefix>1.0.0</VersionPrefix>
+        <VersionSuffix>beta3</VersionSuffix>
```

Comment:
`rc1`

Please also move this file to the `build` folder, so we keep everything in the same place.